<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detail_pemesanan</class>
 <widget class="QDialog" name="detail_pemesanan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>766</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>766</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #6a11cb, stop:1 #2575fc);
</string>
  </property>
  <widget class="QFrame" name="kereta_box">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>1041</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(20, 91, 166, 120); /* Transparansi */
border-radius: 15px; /* Membuat sudut melengkung */
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="box_detail_pemesanan" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>981</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #145ba6; /* Transparansi */
border-radius: 15px; /* Membuat sudut melengkung */</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>921</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="line_nama">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>-10</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>masukan nama anda .... </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_email">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>masukkan email anda .... </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_nomor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>861</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    border: none;
    border-bottom: 2px solid black;
	border-radius: 0px;
    color: black;
    font-size: 16px;
    padding: 5px;
    background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>masukkan No Telpon anda.....</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>YA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tampilkan Sebagai Penumpang </string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="text_detail">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>831</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DETAIL PEMESANAN </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_data_pesan_tiket">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>981</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(156, 209, 255); /* Transparansi */
border-radius: 15px; /* Membuat sudut melengkung */</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;PESAN TIKET &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_keluar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>981</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(44, 181, 255);
    color: white;
    border-radius: 10px;
	font-family: 'Montserrat';
    font-weight: bold;
    font-size: 14pt; 
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
    </property>
    <property name="text">
     <string>KELUAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_lanjutkan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>981</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(44, 181, 255);
    color: white;
    border-radius: 10px;
	font-family: 'Montserrat';
    font-weight: bold;
    font-size: 14pt; 
    padding: 8px;
}
QPushButton:hover {
    background-color: #0056b3;
}
</string>
    </property>
    <property name="text">
     <string>LANJUTKAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_waktu_ticket">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>     tanggal dan waktu keberangkatan </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nama_kereta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>NAMA KERETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_layanan">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>JENIS LAYANAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_penumpang">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1 Penumpang</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_asal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ASAL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tujuan">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TUJUAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>→ </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_harga">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>HARGA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1611</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>kereta_box</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
